.github/workflows/rust.yml

```
@@ -0,0 +1,33 @@
+    strategy:
+      matrix:
+        app-dir: [hello-koma, actix-gcd]  
+    defaults:
+      run:
+        working-directory: ${{ matrix.app-dir }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)